pkg/kubernetes/config/v1beta2/kubeadm.go

```diff
@@ -63,7 +63,9 @@ networking:
   serviceSubnet: {{ .ServiceSubnet }}
 apiServer:
   extraArgs:
+    {{- if .InternalLBDisabled }}
```

Comment:
`advertise-address` shouldn't exist here，this will result in only one api-endpoint.